tests/back/api/test_dataset_api.py

```diff
@@ -92,17 +88,25 @@ def test_get_wrong_dataset(client: TestClient):
     assert response.text == '{"detail":"Dataset not found"}'
 
 
-def test_modify_dataset(client: TestClient):
+def test_get_types(client: TestClient):
+    response = client.get("/api/v1/dataset/types/2")
+    data = response.json()
+    assert data["SepalLengthCm"]["type"] == "Value"
+    assert data["SepalLengthCm"]["dtype"] == "float64"
+    assert data["Species"]["type"] == "Value"
+    assert data["Species"]["dtype"] == "string"
+
+
+def test_modify_dataset_name(client: TestClient):
     response = client.patch(
         "/api/v1/dataset/2",
-        params={"name": "test_modify_name", "task_name": "UnknownTask"},
+        data=json.dumps({"name": "test_modify_name"}),
```

Comment:
Why to use here json.dumps?

Reply:
Since we are using pydantic models to valite the parameters is convenient to send a JSON to the end-point, however I changed it to use json parameter instead of the data one to clean up the code.
```suggestion
        json={"name": "test_modify_name"},
```